### PR TITLE
Add docs about using pbuilder.

### PR DESCRIPTION
**Pull request by mkohler**
I'm not even sure whether using pbuilder and dh-virtualenv together is a good idea, but here are some tips that I've gathered from experience.

### CODE REVIEW

File: doc/usage.rst [hunk 1]
```diff
@@ -239,6 +239,51 @@ additional Python Package Index URI:
...
+``requirements.txt`` in the virtualenv. This option is especially important if
+you are using the --system-site-packages option in your virtualenv.
+
+Here's an example of how to use these arguments in your ``rules`` file.
+
+.. code-block:: make
+
+                override_dh_virtualenv:
+                	dh_virtualenv \
+                	--use-system-packages \
```
Comment (nailor) [inline doc/usage.rst]: Maybe drop this line? It shouldn't be needed and might just cause people accidentally copy-pasting this into their builds

**Comment by nailor** [comment]
Really nice! Thanks for the documentation. I got small fix 

And sorry for late reviews

**Comment by mkohler** [comment]
The Travis build has been "in progress" for over 12 hours.

**Comment by Infinity80880823** [comment]
A.payment.Cola.aBTC*99999999999.464³³³³³BTC

**Comment by nailor** [comment]
Thanks for fixes, looks like Travis is done too. Merging 👍

**Pull request merged**